docs/docs/guides/labs/components/building-pipelines-with-components/adding-components.md

```diff
@@ -68,9 +68,28 @@ Each `component.yaml` file supports a rich templating syntax, powered by `jinja2
 A common use case for templating is to avoid exposing environment variables (particularly secrets) in your YAML files. The Jinja scope for a `component.yaml` file contains an `env` function that can be used to insert environment variables into the template:
 
 ```yaml
-component_type: my_snowflake_component
+type: snowflake_lib.SnowflakeComponent
 
 attributes:
   account: "{{ env('SNOWFLAKE_ACCOUNT') }}"
   password: "{{ env('SNOWFLAKE_PASSWORD') }}"
+```
+
+#### Multiple documents within a file
+
+`component.yaml` can contain multiple instances. If you add a separator `---`, it will evaluate another block of yaml that has with `type` and `attributes` at the top-level.
```

Comment:
I'd phrase as "To configure multiple instances of a component in the same `component.yaml` file, add another block of YAML with top-level `type` and `attributes` keys, separated from the previous block by the `---` separator" (rework as you see fit)